src/bugbug/data/requirements.nix

```diff
@@ -307,14 +307,15 @@ let
     "bugbug" = python.mkDerivation {
       name = "bugbug-0.0.1";
       src = pkgs.fetchurl {
-        url = "https://github.com/marco-c/bugbug/archive/b5d33dea7e6a7249602c1d1b58f26bb4e1e77183.tar.gz";
-        sha256 = "63e8dc33ce21f78e710289dce0c07f430ac551bfd75a68d141b58a4174785036";
+        url = "https://github.com/marco-c/bugbug/archive/c3f68128e9f7a7a142d88d80bfd7e42477f0fbf7.tar.gz";
+        sha256 = "476421dbb014b88cb15643dd4a342027d9f68c363ce2a6aef874d90d18d7aeac";
       };
       doCheck = commonDoCheck;
       checkPhase = "";
       installCheckPhase = "";
       buildInputs = commonBuildInputs ++ [ ];
       propagatedBuildInputs = [
+        self."en-core-web-sm"
```

Comment:
Even though I have this dependency in bugbug now, I still need to manually add `https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-2.0.0/en_core_web_sm-2.0.0.tar.gz#egg=en-core-web-sm` in requirements.txt. pypi2nix fails otherwise when updating depedencies (tries hitting https://pypi.org/ even though the package is not uploaded to pypi).

CC @garbas

Reply:
not sure what you are asking here. for me it looks like `en-core-web-sm` is a dependency of `spacy` since you need to create a symlink there.